project/PACKAGERS.md

```diff
@@ -123,40 +77,17 @@ from the upstream Golang perspective.
 
 ## Building Docker
 
-Please use our build script ("./hack/make.sh") for all your compilation of
-Docker. If there's something you need that it isn't doing, or something it could
-be doing to make your life as a packager easier, please get in touch with Tianon
-and help us rectify the situation. Chances are good that other packagers have
-probably run into the same problems and a fix might already be in the works, but
-none of us will know for sure unless you harass Tianon about it. :)
-
-All the commands listed within this section should be run with the Docker source
-checkout as the current working directory.
-
-### `AUTO_GOPATH`
```

Comment:
Do we still have the code for the AUTO_GOPATH, or was it removed?

I recall seeing the variable recently, but don't recall where 🤔

Reply:
Yeah, we do still have the code for it: https://github.com/moby/moby/blob/0ad2293d0e5bbf4c966a0e8b27c3ac3835265577/hack/make.sh#L67-L78

However, in Go 1.17+ we have to migrate completely off `GOPATH`, and I don't think many packagers will _actually_ be using that functionality, so I figured it was a good time to remove it here at least.